ssl/ssl_transfer_asn1.cc

```diff
@@ -731,3 +1202,21 @@ int SSL_to_bytes(const SSL *in, uint8_t **out_data, size_t *out_len) {
 
   return CBB_finish(cbb.get(), out_data, out_len);
 }
+
+bool SSL3_STATE_get_optional_asn1_int64(CBS *cbs, int64_t *out,
```

Comment:
Would this be worth writing into a new CBS function called `CBS_get_optional_asn1_int64`? I'm also curious as to why `ticket_age_skew` needs a `int64` while other fields use a `uint64_t`?

Reply:
Since this is the difference between the client and server ticket age it can be a negative value.